openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsSlider/SuggestionsSlider.tsx

```diff
@@ -33,39 +35,47 @@ const SuggestionsSlider = () => {
       </Typography.Text>
       <AvatarCarousel />
       {selectedUserSuggestions.length > 0 && (
-        <>
+        <Space className="slider-btn-container m-l-xss">
           <Button
+            ghost
             data-testid="accept-all-suggestions"
+            icon={<CheckOutlined />}
             loading={loadingAccept}
-            size="small"
             type="primary"
             onClick={() =>
               acceptRejectAllSuggestions(
                 SuggestionType.SuggestDescription,
                 SuggestionAction.Accept
               )
             }>
-            <Typography.Text className="text-xs text-white">
+            <Typography.Text className="text-xs text-primary font-medium">
               {t('label.accept-all')}
             </Typography.Text>
           </Button>
           <Button
             ghost
             data-testid="reject-all-suggestions"
+            icon={<CloseOutlined />}
             loading={loadingReject}
-            size="small"
             type="primary"
             onClick={() =>
               acceptRejectAllSuggestions(
                 SuggestionType.SuggestDescription,
                 SuggestionAction.Reject
               )
             }>
-            <Typography.Text className="text-xs text-primary">
+            <Typography.Text className="text-xs text-primary font-medium">
               {t('label.reject-all')}
             </Typography.Text>
           </Button>
-        </>
+          <Button
+            ghost
+            className="exit-suggestion"
+            data-testid="close-suggestion"
+            icon={<CloseOutlined />}
+            onClick={() => onUpdateActiveUser()}
```

Comment:
```suggestion
            onClick={onUpdateActiveUser}
```

Reply:
onUpdateActiveUser requires a parameter of type undefined or EntityRef, so need like this